scripts/processWork.zs

```diff
@@ -327,6 +327,27 @@ function workEx(machineNameAnyCase as string, exceptionsAnyCase as string,
       // mods.ic2.ThermalCentrifuge.addRecipe([IItemStack[] outputs, IIngredient input, @Optional int minHeat);
       mods.ic2.ThermalCentrifuge.addRecipe(outputItems, inputIngr0);
       return machineName;
+    } //Mass Spectrometer dirty gem no liquid
+    if (machineName == 'MassSpectrometer') {
```

Comment:
Everywhere except `workEx` function, machine name can be `anyCase`. But there, when code do comparison it must be all lowercase.